DesktopUI2/DesktopUI2/Models/Settings/MultiSelectBoxSetting.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Avalonia.Controls;
+using Avalonia.Controls.Selection;
+using DesktopUI2.Views.Settings;
+using ReactiveUI;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Reactive;
+using System.Windows.Input;
+
+namespace DesktopUI2.Models.Settings
+{
+  public class MultiSelectBoxSetting : ReactiveObject, ISetting
+  {
+    public string Type => typeof(MultiSelectBoxSetting).ToString();
+    public string Name { get; set; }
+    public string Slug { get; set; }
+    public string Icon { get; set; }
+    public string Description { get; set; }
+    public ConnectorBindings Bindings { get; set; }
+    public static List<string> CategoryNames { get; } = new List<string>{"StructuralFraming", "Walls"};
+    public string CurrentCategoryName { get; set; } = string.Empty;
+    public List<string> Values { get; set; }
+    private bool isSearching = false;
+
+    //public ObservableCollection<string> SearchResults { get; } = new ObservableCollection<string>(CategoryNames);
+    //private List<string> _searchResults2 = new List<string> { "StructuralFraming", "Walls" };
+    //public List<string> SearchResults2
+    //{
+    //  get => _searchResults2;
+    //  set => this.RaiseAndSetIfChanged(ref _searchResults2, value);
+    //}
```

Comment:
We can remove these comments if not needed

Reply:
I left some commented code in there because it was part of the more advanced functionality that I was working on, but yeah, you're right; I got rid of it.